daemon/cmd/daemon_main.go

```diff
@@ -253,9 +253,17 @@ func init() {
 	flags.StringSlice(option.DebugVerbose, []string{}, "List of enabled verbose debug groups")
 	option.BindEnv(option.DebugVerbose)
 
-	flags.StringSliceP(option.Device, "d", []string{}, "List of devices facing cluster/external network for attaching bpf_netdev (first device should be one used for direct routing if tunneling is disabled)")
+	flags.StringP(option.Device, "d", "",
+		fmt.Sprintf("Device facing cluster/external network for attaching bpf_host. Deprecated in favor of --%s", option.Devices))
+	flags.MarkDeprecated(option.Device, "This option will be removed in v1.10")
 	option.BindEnv(option.Device)
 
+	flags.StringSlice(option.Devices, []string{}, "List of devices facing cluster/external network for attaching bpf_host")
+	option.BindEnv(option.Devices)
```

Comment:
I would drop the `bpf_host`. A user doesn't know what bpf_host means here. I'd just say `List of devices facing cluster/external network`.

Reply:
Maybe I would add `(used for BPF NodePort, host firewall and BPF masquerading)` to make it clear.

Reply:
I would make it all explained in detail in the doc, but it's probably also too much context for here.